### ISSUE TITLE
dyld: Library not loaded: @rpath/libswiftAVFoundation.dylib

### ISSUE THREAD

**Issue by jokjok** [CosmicMind/Material#726]
dyld: Library not loaded: @rpath/libswiftAVFoundation.dylib
  Referenced from: /Users/martin/Library/Developer/CoreSimulator/Devices/E19F771B-6A6F-4DEE-B0E7-74FBAE299565/data/Containers/Bundle/Application/1E7B1695-9018-4854-87EF-5E8AF8D0C6A7/test.app/Frameworks/Material.framework/Material
  Reason: image not found

The App compiles without errors, and it works on real devices.
On any simulator, instead, as the App opens there is the error above.
Material.framework is in the embedded binaries, and it has been generated with Carthage.
I'm on Xcode 8.2.1 and Swift 3.

**Comment by danieldahan** [comment]
Have you tried including this library? `AVFoundation` ? Are you using the `Capture` component in Material?

**Comment by jokjok** [comment]
Hello Daniel, yes I've tried to add AVFoundation, both as an embedded binary and as a linked one.
And no, I'm not using the Capture component. I'm using Material for the fab menus and colors only, if I recall correctly.

I suspect that this may be some Xcode-related issue.

**Comment by danieldahan** [comment]
Yes, I have seen issues like this go away once the user's DerivedData directory is deleted and all is cleared. Something could be stale, which is causing this issue. Would you mind trying to clean everything up and deleting the user's DerivedData directory?

**Comment by jokjok** [comment]
Just did it, with no luck :(.

**Comment by danieldahan** [comment]
Not sure then. I don't use Carthage. Can you send me a sample project that I can try reproducing this with?

**Comment by jokjok** [comment]
Ok, so, I managed to fix the issue somehow.

I tried making a new empty project and including the Material framework, and the same error appeared.
Then, I tried to run Carthage to fetch the latest version 2.6.0 of Material and then I've included it in the empty project, and it worked.

I then rebuilt with Carthage the version that I needed to avoid other compilation errors in my project (2.4.12), and now it works on both real devices and simulators.

There was something messed up in the build of Material that I had, or it was an incompatibility of that specific version with Xcode 8.2.1 (I don't remember the version of Material that I was using, unfortunately).

Hope this can help somebody else.
Thank you very much for your support.
